lib/livebook/storage/ets.ex

```diff
@@ -72,10 +76,18 @@ defmodule Livebook.Storage.Ets do
   end
 
   @impl GenServer
-  def init(_opts) do
-    table = :ets.new(@table_name, [:named_table, :protected, :duplicate_bag])
+  def init(opts) do
```

Comment:
```suggestion
  def init(opts) do
    # Make sure that this process does not terminate abruptly
    # in case it is persisting to disk. terminate/2 is still a no-op.
    Process.flag(:trap_exit, true)

```